Vizzuality/half-earth-v3
# Feature/aoi history

## Areas of Interest local history
### Description
This PR adds a modal to display the history of previously visited areas of interest avoiding the tool to trigger a new analysis on the servers.
The functionality to replicate this data on the argis online servers, to allow users to share the AOIs with out the need of re running the analysis again, will be delivered in a different PR, once the species data provided by the services is approved.
### Designs
https://www.figma.com/file/TLCNyRujxrUEhpCkNOXA6s/HalfEarth_Map_visual_V02?node-id=1329%3A15271
### Testing instructions
Draw some areas to analyse in order to feed some data into the historic local database. 
Check the AOI history modal and try changing name, deleting, sharing and visiting an already analysed area.
We are not yet storing the areas data in the server data base. This functionality will be developed once the services data returns all the vertebrates. 
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-157
